Modified Log in to azure mentioned twice

taswar opened the pull request:
Modified Log into Azure to Sign in to Azure and removed the duplicated sentence to more meaningful rather than duplication.

azurecla commented:
Hi __@jurajsucik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

taswar committed:
Modified Log in to azure mentioned twice

Modified Log into Azure to Sign in to Azure and removed the duplicated sentence to more meaningful rather than duplication.

GitHubber17 committed:
Update functions-create-first-azure-function.md
